doc/source/user_guide/scale.rst

```diff
@@ -249,7 +249,9 @@ We can use Dask's ``read_parquet`` function, but provide a globstring of files t
 
    import dask.dataframe as dd
 
-   ddf = dd.read_parquet("data/timeseries/ts*.parquet", engine="pyarrow")
+   ddf = dd.read_parquet(
+       "data/timeseries/ts*.parquet", engine="pyarrow", auto_mkdir=True
+   )
```

Comment:
Another option is to set `:okwarning:` for now, as I suppose this is something to be fixed on the dask side